adam_core/coordinates/transform.py

```diff
@@ -1306,14 +1310,14 @@ def transform_coordinates(
         if frame_out != "equatorial" and frame_out != "ecliptic":
             raise ValueError("".join(frame_err).format("frame_out"))
     else:
-        frame_out = coords.frame
+        frame_out = str(coords.frame.name[0])
 
-    origin_err = ["{} should be one of:\n", "'heliocenter' or 'barycenter'"]
+    origin_err = ["{} should be one of:\n", "'SUN' or 'SOLAR_SYSTEM_BARYCENTER'"]
     if origin_out is not None:
-        if origin_out != "barycenter" and origin_out != "heliocenter":
+        if origin_out != "SOLAR_SYSTEM_BARYCENTER" and origin_out != "SUN":
             raise ValueError("".join(origin_err).format("origin_out"))
```

Comment:
Why is this error getting generated with string formatting, instead of being a literal?